packages/create-hydrogen/src/services/init.ts

```diff
@@ -164,6 +164,24 @@ async function init(options: InitOptions) {
           task.title = 'Completed clean up'
         },
       },
+      {
+        title: 'Initializing a Git repository...',
+        task: async (_, task) => {
+          await git.initializeRepository(templateScaffoldDir)
+          try {
+            await git.commitAll('Initial Commit', {directory: templateScaffoldDir})
```

Comment:
I don't think we should be adding our own commit like this. I know this is common when setting up a repository for someone but that's because we have to. In my experience of running these kinds of commands it's just doing `git init` and letting you handle the rest. 

This is particularly important if there are some things being committed that shouldn't be (e.g. `node_modules`).

Reply:
👍🏻 sounds good, if that's the pattern that things are going these days then let's follow suit! I just did a `create-react-app` and it came with a `.gitignore` of this:

```
# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.

# dependencies
/node_modules
/.pnp
.pnp.js

# testing
/coverage

# production
/build

# misc
.DS_Store
.env.local
.env.development.local
.env.test.local
.env.production.local

npm-debug.log*
yarn-debug.log*
yarn-error.log*
```